ci/yocto-check-layer.sh

```diff
@@ -1,4 +1,6 @@
 #!/bin/sh -e
+# Copyright (c) 2024 Qualcomm Innovation Center, Inc. All rights reserved.
+# SPDX-License-Identifier: MIT
```

Comment:
We can exclude such trivial shell scripts, but it's probably easier to add the markings.